packages/react-select/src/Select.tsx

```diff
@@ -1192,6 +1198,7 @@ export default class Select<
     ) {
       return;
     }
+    this.dropdownEventAlreadyHandled = true;
```

Comment:
Makes sense. I'll update accordingly.

Reply:
Yes, this will not open the menu.
However, clicking on the Clear indicator behaves the same as clicking on the Dropdown indicator without the changes in this PR.

Furthermore, "the same" meant that propagation in general should be allowed, not this implementation specifically.

Reply:
Done. Do you think it would make sense to remove `stopPropagation` from two other places:
- `onMouseDown` in `isMulti` variant,
- `onMenuMouseDown` handler?

Reply:
`onMouseDown` for `MultiValue` component, yes.

`onMenuMouseDown` does not bubble up to the control. I would exclude it for now and revisit it later if the need arises.